addon/components/frost-toggle-button.js

```diff
@@ -0,0 +1,106 @@
+import Ember from 'ember'
+const {computed, Component, ViewUtils} = Ember
+import layout from '../templates/components/frost-toggle-button'
+import FrostEventsProxy from '../mixins/frost-events-proxy'
+
+export default Component.extend(FrostEventsProxy, {
+
+  // ==========================================================================
+  // Properties
+  // ==========================================================================
+
+  layout: layout,
+
```

Comment:
Just a code style thing, but can you alphabetize the properties and remove the lines between them?